lmfdb/cluster_pictures/test_clusterpicture.py

```diff
@@ -0,0 +1,88 @@
+""" Still todo
+# -*- coding: utf-8 -*-
+from lmfdb.tests import LmfdbTest
+
+class NumberFieldTest(LmfdbTest):
```

Comment:
It looks like `test_clusterpicture.py` got copied from number fields and not edited.

Reply:
Yes, a lot has been copied from the number fields case. I was not sure about good tests yet, so I just left it like this. I will think again about some test to do.